include/grpcpp/ext/otel_plugin.h

```diff
@@ -57,6 +58,27 @@ class OpenTelemetryPluginOption {
 /// grpc.server.call.rcvd_total_compressed_message_size
 class OpenTelemetryPluginBuilder {
  public:
+  /// Configuration (scope) for a specific client channel.
+  class ChannelScope {
```

Comment:
Instead of duplicating the existing class, let's just use an alias for the underlying type.  I'm thinking something like this:

```
using grpc_core::StatsPlugin::ChannelScope;
```

I realize that this means exposing the stats plugin APIs from C-core, but I think that's the right thing to do here anyway.

Reply:
Exposing the stats plugin APIs from C-Core and directly using it from public APIs can be bad since those internal Core APIs can change without gRFCs.

Additionally, `grpc_core::StatsPlugin::ChannelScope` is defined in a `src/core` header which is not installed, so that won't work.

Reply:
done as discussed offline